tensorflow/lite/micro/micro_interpreter_test.cc

```diff
@@ -84,6 +84,7 @@ TF_LITE_MICRO_TEST(TestInterpreter) {
     TF_LITE_MICRO_EXPECT_LE(interpreter.arena_used_bytes(), 928 + 100);
     TF_LITE_MICRO_EXPECT_EQ(static_cast<size_t>(1), interpreter.inputs_size());
     TF_LITE_MICRO_EXPECT_EQ(static_cast<size_t>(2), interpreter.outputs_size());
+    TF_LITE_MICRO_EXPECT_GT(interpreter.tensors_size(), static_cast<size_t>(0));
```

Comment:
do you think we could we make this more specific? maybe have the check be EXPECT_EQ based on what we expect from the simple mock model?

Reply:
Done.